Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -4,34 +4,287 @@
+
+.. tabs::
+
+    .. group-tab:: GKE
+
+       The following command creates a Kubernetes cluster using `Google
+       Kubernetes Engine <https://cloud.google.com/kubernetes-engine>`_.  See
+       `Installing Google Cloud SDK <https://cloud.google.com/sdk/install>`_
+       for instructions on how to install ``gcloud`` and prepare your
+       account.
+
+       .. code-block:: shell-session
+
+           export NAME="$(whoami)-$RANDOM"
+           gcloud container clusters create "${NAME}" --zone us-west2-a 
+           gcloud container clusters get-credentials "${NAME}" --zone us-west2-a
+
+    .. group-tab:: AKS
+
+       The following command creates a Kubernetes cluster using `Azure
+       Kubernetes Service <https://docs.microsoft.com/en-us/azure/aks/>`_. See
+       `Azure Cloud CLI
+       <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>`_
+       for instructions on how to install ``az`` and prepare your account.
+
+       .. code-block:: shell-session
+
+           export NAME="$(whoami)-$RANDOM"
+           export AZURE_RESOURCE_GROUP="aks-cilium-group"
+           az group create --name "${AZURE_RESOURCE_GROUP}" -l westus2
+           az aks create --resource-group "${AZURE_RESOURCE_GROUP}" --name "${NAME}" --network-plugin azure
+           az aks get-credentials --name "${NAME}" --resource-group "${AZURE_RESOURCE_GROUP}"
+
+       .. attention::
+
+           Do NOT specify the ``--network-policy`` flag when creating the
+           cluster, as this will cause the Azure CNI plugin to install unwanted
+           iptables rules.
+
+    .. group-tab:: EKS
+
+       The following command creates a Kubernetes cluster with ``eksctl``
+       using `Amazon Elastic Kubernetes Service
+       <https://aws.amazon.com/eks/>`_.  See `eksctl Installation
+       <https://github.com/weaveworks/eksctl>`_ for instructions on how to
+       install ``eksctl`` and prepare your account.
+
+       .. code-block:: shell-session
+
+           export NAME="$(whoami)-$RANDOM"
+           eksctl create cluster --name "${NAME}" --region eu-west-1 --without-nodegroup
+
+    .. group-tab:: kind
+
+       Install ``kind`` >= v0.7.0 per kind documentation:
+       `Installation and Usage <https://kind.sigs.k8s.io/#installation-and-usage>`_
+
+       .. parsed-literal::
 
+          curl -LO \ |SCM_WEB|\/Documentation/gettingstarted/kind-config.yaml
+          kind create cluster --config=kind-config.yaml
+
+    .. group-tab:: minikube
+
+       Install minikube >= v1.5.2 as per minikube documentation: 
+       `Install Minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_.
+
+       .. code-block:: shell-session
+
+          minikube start --network-plugin=cni
 
 Install Cilium
 ==============
 
-.. parsed-literal::
+You can install Cilium on any Kubernetes cluster. If you already have a
+Kubernetes cluster, skip this step. If you want to create a cluster, pick one
+of the options below:
```

Comment:
Just noticed the bit about skipping the step if a cluster is already available should probably be moved to the previous section.